yarn.lock

```diff
@@ -2,52 +2,46 @@
 # yarn lockfile v1
 
```

Comment:
@langolf - let's not run yarn upgrade in a feature PRs please - let's do it in a separate PR